src/vcl/FaissDescriptorSet.cc

```diff
@@ -289,7 +289,11 @@ FaissIVFFlatDescriptorSet::FaissIVFFlatDescriptorSet(
   // TODO: Revise nlist param for future optimizations.
   // 4 is a suggested value by faiss for the IVFFlat index,
   // that's why we leave it for now.
-  int nlist = 4;
+  // int nlist = 4;
+
+  // default value of 4 is too low for any sizeable dataset
+
+  int nlist = 16;
```

Comment:
Suggestion to replace lines 291-296 with:
```
  // default value of 4 is too low for any sizeable dataset

  int nlist = 16;
```
We may want to add this as a parameter in the future. Is there a reason 16 generally works best?

Reply:
added a comment in source code. I agree on we need to define an API for VDMS to set runtime parameters (search parameters) of all the VCL indices in future releases